stable/chartmuseum/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Host your own Helm Chart Repository
 name: chartmuseum
-version: 2.6.1
+version: 2.6.2
```

Comment:
Need a minor version bump here as we are adding a feature

Reply:
Bumped to `2.7.0`